packages/nocodb/src/utils/acl.ts

```diff
@@ -330,6 +331,7 @@ const rolePermissions:
       userUpdate: true,
       userDelete: true,
       generateResetUrl: true,
+      webhookPluginList: true,
```

Comment:
This might make the plugin config accessible to users with the Creator role.

cc @dstala

Reply:
The webhook_url and channel name are stored along with webhook config. So even if a superAdmin created a webhook with slack/teams, the webhook_url would be accessible to base_creator.

Reply:
For me, user with creator access should be able to create webhook. if there is a way not to expose URL & just let user select from the dropdown - it would be preferred.

Reply:
> if there is a way not to expose URL & just let user select from the dropdown - it would be preferred.

We dont have any unique id per channel and we do not have the channel name as unique. So we won't be able to drop the webhook_url.